schemas/schema-schema.ipldsch

```diff
@@ -370,13 +370,13 @@ type StructRepresentation union {
 } representation keyed
 
 ## StructRepresentation_Map describes a way to map a struct type onto a map
-## representation.  For example, fields may be aliased,
-## or implicit values associated.
+## representation.  For example, fields may be renamed, or implicit values
```

Comment:
How's this sound:

> For example, the keys that fields are emitted under may be remapped using the 'rename' instruction,

Reply:
nah, don't like that either, how about this:

> Field serialization options may optionally be configured to enable mapping serialized keys using the 'rename' option, or implicit values specified where the field is omitted from the serialized form using the 'implicit' option.